### DISCUSSION TITLE
How do I run Visual Studio xamarin gtk# apps on WSL2/Ubuntu ?

### DISCUSSION THREAD

**Discussion by ekrogh** [xamarin/Xamarin.Forms#11669] [General]
# Description
I'd just like to ask: How do I run/test a GTK# app, created with Visual Studio 2019 Windows/mac, in Ubuntu 20.04 in wsl2 or Hyper-v ?

-- Eigil

**Comment by PureWeen** [comment]
@jsuarezruiz
